### DISCUSSION TITLE
Support diffs Jupyter Notebooks (.ipynb files)

### DISCUSSION THREAD

**Discussion by KasperZutterman** [community/community#5024] [Other Feature Feedback, Questions, & Ideas]
Jupyter notebooks are hard to version control.

It would be nice to have an easy way to diff these notebooks

**Comment by tkw1536** [comment]
There is https://github.com/jupyter/nbdime. Not sure if this is something that should be integrated into GitHub directly.

**Reply by KasperZutterman** [reply]
I am aware of the nbdime tool, but in my opinion, it would be nice to have a rendered view of the cells where the diffs are shown, instead of how it is now when you make a change: 
![image](https://user-images.githubusercontent.com/21260838/129472618-d0ec7ec3-4b2d-4d9d-b27e-aea14855819a.png)
